implementations/rust/ockam/ockam_identity/src/identities/identities_creation.rs

```diff
@@ -159,6 +142,7 @@ impl IdentitiesCreation {
             return Err(IdentityError::WrongSecretKey.into());
         }
 
+        self.repository.store_change_history(&identity).await?;
```

Comment:
I think it's part of `self.update_identity` call few lines above. This line can be removed

Reply:
Done